src/scripts.js

```diff
@@ -52,4 +54,34 @@ const findDestinationForTrip = (trip, allDestinationData) => {
   return allDestinationData.find((destination) => destination.id === trip.destinationID);
 }
 
-window.addEventListener('load', fetchAll);
\ No newline at end of file
+window.addEventListener('load', fetchAll);
+
+const addNewTripButton = document.querySelector('#addNewTripButton');
```

Comment:
I like how you combined your query selectors with your event listeners/callback functions in an easy to read and track format